product_docs/docs/pgd/3.7/bdr/durability.mdx

```diff
@@ -44,6 +44,12 @@ Replication](https://www.postgresql.org/docs/11/warm-standby.html#SYNCHRONOUS-RE
 (PSR), which is uni-directional, but offers a synchronous variant that
 can used in combination with BDR.
 
+!!! WARNING
+    While using multiple BDR enabled databases (we advise against it)
+    the lsn sync confirmation may come from a diferent database on a
+    node that is in our synchronous_standby_names and thus not assure
+    that the data is really synced on the database that we really want.
+
```

Comment:
This is not as clear as it should be, IMO. I also think we should avoid "we" in documentation, but use third person.

How about this:
This only works when using a single database per node. When using multiple BDR enabled databases per node, which is not generally recommended, the LSN based confirmations may originate from any one of the databases from a node specified in `synchronous_standby_names` and thus not assure the data is really flushed to disk.

Reply:
Done, thanks! 

@drothery-edb could you please check/merge it?